### PR TITLE
Add unit test for microservices connector router

### PR DESCRIPTION
**Pull request by zhlsunshine**
## Description

Change List:

1. Add unit test for microservices connector router
2. Replace `gmc` into `mc` for microservices connector CRD api, etc.  [Issue#48](https://github.com/opea-project/GenAIInfra/issues/48)

## Issues

https://github.com/opea-project/GenAIInfra/issues/48

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

n/a

## Tests

The unit test is for microservices connector router.

### CODE REVIEW

File: microservices-connector/cmd/router/main_test.go [hunk 1]
```diff
@@ -0,0 +1,560 @@
...
+	expectedResponse := map[string]interface{}{
+		"predictions": "2",
+	}
+	fmt.Printf("final response:%v\n", response)
+	assert.Equal(t, expectedResponse, response)
+}
+
+func TestSimpleModelEnsemble(t *testing.T) {
+	// Start a local HTTP server
+	model1 := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
```
Comment (irisdingbj) [inline microservices-connector/cmd/router/main_test.go:125]: change all 'model' to 'service' : eg: model1 -> service1

File: microservices-connector/cmd/router/main_test.go [hunk 2]
```diff
@@ -0,0 +1,560 @@
...
+		},
+		"model2": map[string]interface{}{
+			"predictions": "2",
+		},
+	}
+	fmt.Printf("final response:%v\n", response)
+	assert.Equal(t, expectedResponse, response)
+}
+
+func TestInferenceGraphWithCondition(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (zhlsunshine) [reply microservices-connector/cmd/router/main_test.go:229]: thanks, done!

File: microservices-connector/cmd/router/main_test.go [hunk 3]
```diff
@@ -0,0 +1,560 @@
...
+		return
+	}
+	expectedResponse := map[string]interface{}{
+		"predictions": "2",
+	}
+	fmt.Printf("final response:%v\n", response)
+	assert.Equal(t, expectedResponse, response)
+}
+
+func TestSimpleModelEnsemble(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (zhlsunshine) [reply microservices-connector/cmd/router/main_test.go:123]: thanks done!

File: microservices-connector/cmd/router/main_test.go (hunk 1 above)
Comment (zhlsunshine) [reply microservices-connector/cmd/router/main_test.go:125]: thanks, done!

**Review by irisdingbj — APPROVED** [review]

**Review by mkbhanda — APPROVED** [review]
Thank you for the unit and end to end tests and changing the version name.

**Pull request merged**